client/src/pages/NewGame.js

```diff
@@ -0,0 +1,28 @@
+/*
+UI for creating a new game.
+*/
+
+import React from "react";
+import GameInvitation from "../components/GameInvitation"
+import { Button } from '@material-ui/core';
+import "../stylesheets/newgame.css";
```

Comment:
Can you move these styles into a makeStyles function in this file?

Reply:
Done